Exercise-13/js/index.js

```diff
@@ -0,0 +1,39 @@
+import data from "../assets/friends.json" assert { type: "json" };
+
+const cardsContainer = document.getElementById("cardsContainer");
+
+function addFriendsData() {
+  const result = new DocumentFragment();
+  for (let friendDetail of data) {
+    // image container
+    const friendImageContainer = document.createElement("div");
+    friendImageContainer.classList.add("friend-image");
+    const image = document.createElement("img");
+    image.setAttribute("src", friendDetail.img);
+    image.setAttribute("alt", friendDetail.first_name);
+    friendImageContainer.appendChild(image);
+
+    // detail container
+    const friendDetailsContainer = document.createElement("div");
+    friendDetailsContainer.classList.add("friend-details");
+    const friendName = document.createElement("p");
+    friendName.classList.add("friend-name");
+    friendName.innerText = friendDetail.first_name + " " + friendDetail.last_name;
+    const friendEmail = document.createElement("p");
```

Comment:
could you confirm if email also could be in p tag ?

Reply:
For emails, it is best to use anchor tag. I have changed it.